### PR TITLE
Detect os: Rosa, MRE and Detect browser MAUI WAP Browser and Detect new brands: Xtouch, Fonos, Conquest and Detect devices for existing brands: Bush, Cat, DEXP, Lava, Explay, Artel, Pixus, Acer, Ark, MAXVI, Videocon, MicroMax, Huawei, Asus, Realme, Aligator, Unihertz, NOA, Maxtron, Alcatel, Irbis, Lenovo, Oukitel, Invin, Black Fox, Xiaomi, Ulefone, BQ, Wiko, Vestel, Odys, Kivi, Multilaser, Blackview, Haier, Nomu, Vertex, Coolpad

### PR DESCRIPTION
**Pull request by sanchezzzhak**

### CODE REVIEW

File: regexes/oss.yml [hunk 1]
```diff
@@ -160,6 +160,13 @@
   name: 'MTK / Nucleus'
   version: '$1'
 
+##########
+# MRE / MAUI Runtime Environment
+##########
+- regex: 'MRE/(\d+)\.(\d+).*;MAUI'
+  name: 'MRE'
+  version: '$1.$2'
```
Comment (sanchezzzhak) [inline regexes/oss.yml:168]: `MRE` not sure about the correct name, MRE is more of an SDK (created in Mediatek)

Comment (sgiehl) [reply regexes/oss.yml:168]: Not sure here either. Tried to find some useful information on that, but couldn't :shrug: Other parsing libraries are detecting it as `MRE`, so might be fine.

File: regexes/device/mobiles.yml [hunk 2]
```diff
@@ -14222,15 +14331,19 @@ Wiko:
...
       model: 'View Max'
+    - regex: 'W_P130'
+      model: 'View Go'
     - regex: 'W-V800-(TVM|EEA|OPE)'
       model: 'View 3 Lite'
-    - regex: 'W-K420-EEA'
+    - regex: 'W-K130-(?:TMV|OPE|EEA)'
       model: 'Y50'
-    - regex: 'W-K510-(TVM|EEA|OPE|SUN)'
+    - regex: 'W-K420-EEA'
```
Comment (sgiehl) [inline regexes/device/mobiles.yml:14340]: According to a quick search `W-K420-EEA` is actually the `Y50` as it was defined before. Where did you have the information from that it's `Y70`?

Comment (sanchezzzhak) [reply regexes/device/mobiles.yml:14340]: I used the official website https://world.wikomobile.com/red.php?telephone=2725

Comment (sgiehl) [reply regexes/device/mobiles.yml:14340]: ok fine. Guess the search results on Google are a bit confusing than...

**Review by sgiehl — APPROVED** [review]

**Comment by liviuconcioiu** [comment]
W-K420-EEA is indeed Y50, as @sgiehl said.

`Wiko,Y50,W-K420,W-K420-EEA`

@sanchezzzhak can you please double check? I've added the link with devices that Google Play uses.

[https://storage.googleapis.com/play_public/supported_devices.csv](https://storage.googleapis.com/play_public/supported_devices.csv)

**Pull request merged**